### PR TITLE
Favicon tidy-ups

### PR DESCRIPTION
**Pull request by zerolab**
Based on https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

To-Do:
- [x] finish the svg favicon. Could use some help here @nicklee (bonus points: use `@media (prefers-color-scheme: dark)` for dark mode styling in the same svg)
- [x] export a 512x512 variant to `wagtailio/static/img/favicons/android-icon-512x512.png` (currently it is a copy of the 192x192 one)

Nice to have - revised favicon. currently is using the old `#43b1b0` background. Could probably move to the `#2e1f5e` (as the primary colour in Wagtail 4) or the grey one as in the [default sharing image](https://github.com/wagtail/wagtail.org/blob/next/wagtailio/static/img/default-sharing-image.png)

### CODE REVIEW

**Comment by zerolab** [comment]
@vossisboss would be cool if you could stir up some resources for this, if at all possible
tl;dr -- need a nice SVG logo variant, ideally with a dark scheme version

**Comment by vossisboss** [comment]
@zerolab So just to confirm, we need two variants (light and dark) on a new favicon that can be exported to dimensions of 512 X512?

If that's the case, I might be able to get Ben to slip it in next week when he rotates into his Wagtail design time.

**Comment by zerolab** [comment]
@vossisboss that's right

**Comment by nicklee** [comment]
Images added, including a dark mode. The Favicon is now a closer crop of the wagtail's head.

Screenshot 2023-07-25 at 08 41 08

**Comment by nicklee** [comment]
I've just merged this into the new dark mode MR here: https://github.com/wagtail/wagtail.org/pull/391

Semi linked now we have a dark mode favicon :)
